conftest.py

```diff
@@ -1,18 +0,0 @@
-import os
-
-import pytest
-
-
-@pytest.fixture(scope="session", autouse=True)
-def _temp_cwd(tmpdir_factory):
-    """
-    This fixture creates a temporary current working directory
-    for the test session, so that docstring tests that write files
-    don't clutter up the real cwd.
-    """
-    original_cwd = os.getcwd()
-    try:
-        os.chdir(tmpdir_factory.mktemp("cwd"))
-        yield
-    finally:
-        os.chdir(original_cwd)
```

Comment:
You moved this to the `asdf._tests.conftest` this means it will only be scoped to tests in `asdf._tests` (yes it says session, but the tests in the session it applies to are the ones in its directory or below); however, there are tests in many other directories above this one like the `asdf.commands._tests`. This will also effect running the doctests (if there are any) in `asdf/docs`.

Reply:
Thanks for the catch. I moved that fixture back.